platform/README.md

```diff
@@ -6,3 +6,122 @@ The directory structure is:
 
 - `LANGUAGE/FRAMEWORK/baseline`: copied from TFB with no code changes. Unnecessary files removed.
 - `LANGUAGE/FRAMEWORK/instrumented`: copy of `baseline`, with code changes to add Sentry instrumentation (error and performance monitoring).
+
+## Add New Platform or Framework
+
+Follow the steps below to add new platforms or frameworks to use with the `sentry-sdk-benchmark` tool.
+
+*The command line examples below all use [Z shell](https://en.wikipedia.org/wiki/Z_shell) syntax. You are not required to use `zsh`, as the same outcome can be achieved with the tools of your preference.*
+
+
+1. Clone the [TechEmpower Framework Benchmarks](https://github.com/TechEmpower/FrameworkBenchmarks) repository.
+
+    ```zsh
+    git clone --depth=1 git@github.com:TechEmpower/FrameworkBenchmarks.git
+    ```
+
+    or, alternatively, using HTTPS:
+
+    ```zsh
+    git clone --depth=1 https://github.com/TechEmpower/FrameworkBenchmarks.git
+    ```
+
+    We suggest a shallow clone (`--depth=1`) because the repository history is heavy and takes a considerable amount of time to download.
+
+2. Copy files from TFB to `platform/.../baseline/`.
+
+    Set `LANGUAGE` and `FRAMEWORK` below to your wish, depending on what is available in TFB.
+
+    ```zsh
+    LANGUAGE=Go
+    FRAMEWORK=go-std
+    mkdir platform/${LANGUAGE:l}/${FRAMEWORK:l}/baseline/
+    rsync -avz FrameworkBenchmarks/frameworks/${LANGUAGE}/${FRAMEWORK}/ platform/${LANGUAGE:l}/${FRAMEWORK:l}/baseline/
+    ```
+
+    Note that we use lower case directory names, while TFB uses a mix of cases. The form `${LANGUAGE:l}` is a way to tell `zsh` to output a variable in lower case.
+
+
+3. Delete `*.dockerfile` files, except for one that uses the PostgresQL database.
+
+    The `sentry-sdk-benchmark` tool expects to find and use a single Dockerfile in the app directory, and will automatically provide a PostgresQL database.
+
+4. Create a `config.json` file.
+
+    ```zsh
+    cp platform/python/django/config.json platform/${LANGUAGE:l}/${FRAMEWORK:l}/config.json
```

Comment:
Should we have docs about what the config files contains somewhere?

Reply:
I considered that, but I opted for copying from a "known source" instead of documenting each field because they are a moving target. I believe we might still add things and the docs would easily go out of sync.